ui/src/components/Layout/SideNavigation.vue

```diff
@@ -50,13 +62,42 @@
 </template>
 
 <script>
-import { serverBus } from "../../main";
 
 export default {
+  computed:{
+    getChevronIcon()
```

Comment:
Small thing but for computed properties, you should avoid getter / setter terms because you aren't really doing that. You are describing a new property. Instead of getChevronIcon for example, it should just be chevronIcon.

Reply:
Done